solr/CHANGES.txt

```diff
@@ -103,6 +103,7 @@ Optimizations
 
 * GITHUB#2217: Scale to 10K+ collections better in ZkStateReader.refreshCollectionsList (David Smiley)
 
+* SOLR-17099: snitch does not return spurious tags (Pierre Salagnac)
```

Comment:
Can you author a more understandable CHANGES line?

Reply:
Speaking of which, please make no reference of "snitch"; it's no longer a user concept but is some internal dubious naming choice.

Reply:
Thanks for your feedback, and sorry for the time it took me to get back to this change.
I updated the changelog to something more readable.